README.md

```diff
@@ -136,6 +123,28 @@ This script uses the deduplicated dataset as input and performs 2 quality checks
     2. Change the labels of incorrectly included records to excluded.
 It results in corrected columns for both the subject- and the composite-label.
 
+## Improving DOI retrieval speed
+To speed up the retrieval, you can split the dataset into smaller chunks, run
+the script multiple times, and merge the results.
+
+1. Split the dataset into smaller chunks by running the first halve of the 
+`split_and_merge_input_file.ipynb` script. Within this script is the option
+to set the amount of chunks. If the records aren't split evenly, the last
+chunk might be smaller than the others.
+2. For each chunk, create a copy of the `chunk_0.py` file, and place it in
+the `split` folder. Change the name `chunk_0.py` to `chunk_1.py`,
+`chunk_2.py`, etc, for each created chunk. Within each file, change
+`script_number` = "0" to `script_number` = "1", `script_number` = "2", etc.
```

Comment:
Could you make two steps out of this, or an A and B? I think that might reduce the chance that somebody does not read everything.